src/libs/ActiveClientManager/index.ts

```diff
@@ -45,7 +45,23 @@ Onyx.connect({
     },
 });
 
+let wasTheLeader = false;
+
+/**
+ * The last GUID is the most recent GUID, so that should be the leader
+ */
+const isClientTheLeader: IsClientTheLeader = () => {
+    if (wasTheLeader) {
```

Comment:
Can you please add a comment here explaining why this logic is necessary? Maybe a better name for this variable would be `isPromotingAnotherLeader` or something.

Reply:
Fun fact, var name `isPromotingAnotherLeader` was triggering eslint rule `no-negated-condition`. So I had to use `isPromotingNewLeader`